Test/hofs/ReadsReads.dfy

```diff
@@ -3,25 +3,25 @@
 
 module ReadsRequiresReads {
   ghost function MyReadsOk<A,B>(f : A ~> B, a : A) : set<object?>
-    reads f.reads(a)
+    requires f.requires(a) reads f.reads(a)
```

Comment:
line break

Reply:
Not sure what you mean, can you propose the change as a commit?